facdb/datasets.yml

```diff
@@ -186,6 +186,7 @@ datasets:
   - name: sca_enrollment_capacity
 
   - name: doe_lcgms
+    version: 20220513
```

Comment:
not sure why those changes were included here in this PR? @mbh329

Reply:
ok I reread initial comment now I see why. But not sure they should be included since if it is a patch in order for the build to work we probably don't want to include in develop without a proper PR and real fix

Reply:
Yeah sorry about that, should have just branched another feature branch off this and ran it on that. that was my oversight